optimalAssignment.py

```diff
@@ -1,4 +1,40 @@
+from itertools import combinations
+from itertools import permutaions
```

Comment:
this should be `permutations`

Reply:
also import `Vector2D` and `dist` function.

Reply:
ok